scripts/openmc-update-mgxs.py

```diff
@@ -0,0 +1,232 @@
+#!/usr/bin/env python
```

Comment:
Just curious, but is this new script necessary? How many people will need to use it?